### DISCUSSION TITLE
Boxes Version this Pool

### DISCUSSION THREAD

**Discussion by crazyslakr** [TedJuch/TedTournament#6] [General]
Last year I asked about a Boxes version of the Spreadsheet using the TedTournament function over on the Google thread you had from 2021 and you said you had something in the works, but I never saw anything else on it, any chance you still have this lying around and working?  

Another question how would one go about using Vlookup or Import the data on the Data source page instead of Tedtournament in the below formula to put the data on my Bracket page.  So I would have to call the script all the time?  Any ideas or help would be appreciated.

`=CONCATENATE(TedTournament("Men",2022,64,1,14)," ",TedTournament("Men",2022,64,1,1))`

**Comment by TedJuch** [comment]
@crazyslakr I found a Squares template that I had started last year and made some tweaks. Try it out here: https://docs.google.com/spreadsheets/d/1IUe82ZpMFDMlw1OwNw5b96kQTTk_xSNVnO4jbcV1TnI/copy

If you want to pull in all of the data for the tournament into a tab, create a new tab and put this in A1:
`=importrange("1DyuuT9zPSh9RdzrAF_1bY6HhyuYKckL3E6wr-sGKZTs","'Men 2022 Tournament Data'!A:U")
`
Google magic will keep that source updated and then you can do vlookups using the game number to pull in whichever column you need. You can also change Men to Women and/or the year for any year back to 2014 in that formula to pull in historical data.

Here is an vlookup template you could use. Just replace the [value] with what you are looking for:

`=vlookup("[GameID]","'[name of the tab where you wrote the importrange() above]'!$A:$AU",match("[header of the attribute you want]","'[name of the tab where you wrote the importrange() above]'!$A$1:$AU$1,0),FALSE)`

For example, if I wanted the winning team of game 201 which was Gonzaga I would write:

`=vlookup(201,'Men 2022 Tournament Data'!$A:$AU,match("Winning Team",'Men 2022 Tournament Data'!$A1:$AR1,0),FALSE)`

Let me know if you have any other questions!

**Reply by crazyslakr** [reply]
This was exactly what I needed, very much appreciated for sharing.

**Comment by TedJuch** [comment]
Great!

**Comment by crazyslakr** [comment]
Sorry to bother you again, since this isn't your setup, but is there a way to compare two values in 2 cells, ie: winning and losing of a game and then change the colors based on the scores, I was trying the conditional formatting, but it didn't seem to work on both boxes.

**Reply by crazyslakr** [reply]
I've sort of figured it out using a conditional scale formating based on min/max, but copying the format to the cells individually is kind of a pain but it works.

**Reply by TedJuch** [reply]
I have something similar on my Single Bracket template. There is a cell above the cell where the user picks their winning team. The formula checks to see if that team won the game. If the team did win then the box above their pick says "Yes!", if not then it has the name of the team that won. Then I do conditional formatting on that box to turn it green if it says Yes! otherwise it turns red if there is a value. I'm not sure if that helps because you can't really put extra boxes above individual cells in the squares game.

**Reply by crazyslakr** [reply]
Thanks for the response, I actually took a look at that Single bracket template and was able to figure a way to do it, I just had to make a macro to then copy the formatting over to each set of scores that changed the conditional formatting for the color scale.  Since it seems sheets wouldn't let me highlight all the boxes and run the macro once, I just did it for each manually.

The only thing now I would like to do is make the Bracket page look similar to yours, but that's a lot to do to make it compatible, I think.  Yours looks way cleaner lol
